.github/pull_request_template.md

```diff
@@ -0,0 +1,64 @@
+<!-- PR Title Should match format:
+#{TicketNumber}: Description of Changes
+
+if no ticket number, use `chore:`, `fix:`, `feat:` etc.
+
+Example:
+#123: Add pagination to List Applications endpoint
+-->
+
+## Summary
+
+<!-- High level, short description of work done. 1-2 sentences. -->
+
+### Related Issues
+
+- Paste URL to issue 1
+- Paste URL to issue 2
+
+## Description of Changes
+<!-- Describe the changes in your pull request **per service or package**, providing enough context for reviewers.
+
+Be sure to call out any breaking changes, as well as any special instructions required to run the new code (i.e. New or updated dependencies? `pnpm i`. New migrations to run? `pnpm run migrate-dev`. etc.) 
+
+Add a heading for each app/package that you have contributed changes to and list the changes included.
+-->
+
+<!-- EXAMPLE START
+General description of the changes in your PR and the functionality it adds.
+
+### UI
+- Added a new component `ComponentName` which achieves some functionality.
+  - Description of `ComponentName` and the changes you made to create it
+  - Added package [`package name`](https://link.to/package) to handle something
+
+### Server
+- Added new endpoint `GET /stuff`that does stuff
+
+
+### Special Instructions
+Before running these changes, you will need to install `package name`:
+```
+pnpm i
+```
+EXAMPLE END -->
+
+## Readiness Checklist
+
+- [ ] **Self Review**
+  - I have performed a self review of code
+  - I have run the application locally and manually tested the feature
+- [ ] **Title Format**
+  - The PR title is properly formatted to match the pattern: `#{TicketNumber}: Description of Changes`
```

Comment:
This could be PR Format instead of Title Format, with a second bullet point for ' - Included Link to relevant Issue and PR Description'